plugins/settings-sync/src/com/intellij/settingsSync/config/SettingsSyncEnabler.kt

```diff
@@ -53,9 +53,7 @@ internal class SettingsSyncEnabler {
         updateResult = result
         if (result is UpdateResult.Success) {
           val cloudEvent = SyncSettingsEvent.CloudChange(result.settingsSnapshot, result.serverVersionId, syncSettings)
-          runBlockingCancellable {
-            saveSettings(ApplicationManager.getApplication(), forceSavingAllSettings = true)
-          }
+
```

Comment:
@jaen , when `saveSettings(...)` is called app startup (which happens, if IDE is configured to use settingsSync), I faced  exceptions, because the app is not completely loaded. 
So, I have changed it to force saving only for the situations when it's really necessary, i.e. as pushing new settings to the server

Reply:
I see.

What does this mean for this removal? This place calls `SettingsSyncBridge#initialize` with `InitMode.TakeFromServer` — should I keep the removal, but add a case for this to `if (initMode == InitMode.PushToServer)` in `SettingsSyncBridge#initialize`?

Reply:
That's a good question. If you look into history, there's often a push after initial pull, because some settings change (or added). 
IMHO, this is not critical here, as changed settings will eventually be persisted and pushed back to server. So, my personal preference would be to keep that removed and not to add back. We'll see how it works and may add back later, if necessary